emergence_lib/src/organisms/mod.rs

```diff
@@ -3,24 +3,41 @@
 use bevy::prelude::*;
 
 use self::{
+    life_cycles::LifeCycle,
     sessile::{fungi::FungiPlugin, plants::PlantsPlugin},
     units::UnitsPlugin,
 };
 
+pub mod life_cycles;
 pub mod sessile;
 pub mod units;
 
 /// All of the compone
```

Comment:
This looks like a typo
```suggestion
/// All of the components
```